src/conf.c

```diff
@@ -71,7 +73,7 @@ checkVar(dstrbuf *var)
 {
 	int i;
 
-	for (i = 0; i < MAX_CONF_VARS; i++) {
+	for (i = 0; strcasecmp("", conf_vars[i]) ; i++) {
```

Comment:
Then here it would be a bit more simple and performant to just say:

    for (i = 0; conf_vars[i]; i++) {